README.md

```diff
@@ -170,11 +176,28 @@ After the above command completes, rerun the commands given at [Jupyter Classic
 
 ## Connecting to a graph database
 
+Configuration options can be set using the `%graph_notebook_config` magic command. The command accepts a JSON object as an argument. The JSON object can contain any of the configuration options listed below. The command can be run multiple times to change the configuration. The configuration is stored in the notebook's metadata and will be used for all subsequent queries.
+
+| Configuration Option | Description | Default Value | Type |
+| --- | --- | --- | --- |
+| auth_mode | the authentication mode to use for Amazon Neptune connections | DEFAULT | string |
+| aws_region | the AWS region to use for Amazon Neptune connections | your-region-1 | string |
+| host | the host url to form a connection with | localhost | string |
+| load_from_s3_arn | the ARN of the S3 bucket to load data from [Amazon Neptune only] | | string |
+| port | the port to use when creating a connection | 8182 | number |
+| proxy_host | the proxy host url to route a connection through [Amazon Neptune only]| | string |
+| proxy_port | the proxy port to use when creating proxy connection [Amazon Neptune only] | 8182 | number |
+| ssl | whether to make connections to the created endpoint with ssl or not [True/False] | False | boolean |
+| ssl_verify | whether to verify the server's TLS certificate or not [True/False] | True | boolean |
+| sparql | SPARQL connection object | ``` { "path": "sparql" } ``` | string |
+| gremlin | Gremlin connection object | ``` { "username": "neo4j", "password": "password", "auth": true, "database": null } ```| string |
```

Comment:
The default values for Gremlin and Neo4J config sections are switched around here